eahub/config/settings.py

```diff
@@ -195,8 +195,10 @@
 CRISPY_TEMPLATE_PACK = "bootstrap3"
 
 # Django PWNED Passwords
+PWNED_VALIDATOR_ERROR = "Your password was determined to have been involved in a major security breach in the past: https://haveibeenpwned.com/"
```

Comment:
Can you remove the URL from the text and instead make it into a link to https://haveibeenpwned.com/Passwords? HTML is allowed here.

Reply:
Tried this, HTML wasn't parsed but came out as strings

Reply:
Ah, sorry. Here's how to make it work: https://stackoverflow.com/questions/10706296/how-do-i-input-html-into-the-help-text-of-a-django-form-field

Reply:
Done, required though to break up the form

Reply:
I'm confused, I don't see this.

Reply:
sorry, push didn't go through. It's there now